managed_kafka.sh

```diff
@@ -149,11 +141,25 @@ certgen() {
 
     KAFKA_RESOURCE=$(get ${KAFKA_ID})
     KAFKA_USERNAME=$(echo ${KAFKA_RESOURCE} | jq -r .name)
-    KAFKA_CERT=$(mktemp)
+    CRT_PEM=$(mktemp)
     KAFKA_INSTANCE_NAMESPACE='kafka-'$(echo ${KAFKA_RESOURCE} | jq -r .id  | tr '[:upper:]' '[:lower:]')
-    oc get secret -o yaml ${KAFKA_USERNAME}-cluster-ca-cert -n ${KAFKA_INSTANCE_NAMESPACE} -o json | jq -r '.data."ca.crt"' | base64 --decode  > ${KAFKA_CERT}
-    keytool -import -trustcacerts -keystore truststore.jks -storepass password -noprompt -alias mk${KAFKA_ID} -file ${KAFKA_CERT}
-    rm ${KAFKA_CERT}
+    TRUSTSTORE=truststore.jks
+    TRUSTSTORE_PASSWORD=password
```

Comment:
Exporting the `TRUSTSTORE_PASSWORD` variable would allow the keytool commands to use `-storepass:env TRUSTSTORE_PASSWORD` which might provide a little extra security, but not a huge win.